### ISSUE TITLE
File separator used in Save As Markdown

### ISSUE THREAD

**Issue by moriyas1984** [shd101wyy/markdown-preview-enhanced#474]
Hi. 

This package is very useful to me. I am always grateful to you.
I always write a markdown with charts using Plant-UML by this package.
But the charts is not rendered in GitLab's markdown previewer, so I convert it to an image using the feature "Save As Markdown" in this package.

The package makes an image file and automatically writes it's path in the converted markdown using `\` (bachslash) file separator in Windows, but it does not work in GitLab's markdown previewer because GitLab or Linux uses `/` file separator.
But `/` works normally in Atom (or the package? I don't know that well...), so I think the package should write image file's path using `/`.

In other words, "\" works only in Windows but "/" works in both Window and Linux, if the markdown is rendered by Atom and the package.

How do you think about it?

Thanks.

**Comment by shd101wyy** [comment]
Hi @moriyas1984 

Does the newest version fix this issue for you?

Thanks

**Comment by moriyas1984** [comment]
Sorry I found that the `\` in the line 10 is escaped and edited it.
